app/domain/authentication/authn_k8s/inject_client_cert.rb

```diff
@@ -9,13 +9,14 @@ module AuthnK8s
 
     InjectClientCert ||= CommandClass.new(
       dependencies: {
-        logger:                 Rails.logger,
-        resource_class:         Resource,
-        conjur_ca_repo:         Repos::ConjurCA,
-        kubectl_exec:           KubectlExec,
-        validate_pod_request:   ValidatePodRequest.new,
-        extract_container_name: ExtractContainerName.new,
-        audit_log:              ::Audit.logger
+        logger:                        Rails.logger,
+        resource_class:                Resource,
+        conjur_ca_repo:                Repos::ConjurCA,
+        kubectl_exec:                  KubectlExec,
+        set_file_content_in_container: SetFileContentInContainer.new,
```

Comment:
what does this mean? I'm not sure this naming is the clearest, because just from reading this name I am not sure at all what this does

Reply:
after consulting with @sigalsax i renamed it to `copy_text_to_file_in_container`. is this clear? do you have any suggestions?